{"name": "Неуспешный логин с неверными данными", "status": "passed", "start": 1733246820043, "stop": 1733246820233, "uuid": "9bcaaafb-aa4c-4eb6-bec2-1631bcb5b105", "historyId": "875475173a8d8fcaadc0586d31e090c2", "testCaseId": "875475173a8d8fcaadc0586d31e090c2", "fullName": "tests.test_login.TestLogin#test_login_failure", "labels": [{"name": "feature", "value": "Логин пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "admins-MacBook-Pro.local"}, {"name": "thread", "value": "97552-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}